geometry/AdvSND_geom_config.py

```diff
@@ -1,325 +1,672 @@
-import ROOT as r
+"""Configuration parameters for AdvSND geometry."""
```

Comment:
One thing : how about including the info from the SiSensor header also here. So that we have all det related input params in one place?
This is the way we have in snd and it is quite convenient.

Reply:
Definitely possible. For now, since I'm only using those things in C++ I just put it into a header (I was hoping of putting also the Volume/Node creation there, but it wasn't as simple as I hoped... For now, there is still a lot of duplicate code between the two classes).

In any case, that would be a matter for another PR or at least an issue to keep track of the suggestion.

Reply:
I make an issue regarding this then - I will need the sensor info for the tracking for example